keps/0007-skeleton-generator.md

```diff
@@ -0,0 +1,37 @@
+---
+kep-number: 7
+title: Skeleton generator
+authors:
+  - "@djannot"
+owners:
+  - TBD
+editor: TBD
+creation-date: 2019-04-01
+status: provisional
+---
+
+# Skeleton generator
+
+## Table of Contents
+
+  * [Table of Contents](#table-of-contents)
+  * [Summary](#summary)
+  * [Motivation](#motivation)
+
+Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
+
+## Summary
+
+In order to develop a new framework using KUDO, you always need to create a several `Plans` (`deploy`, `update`, `upgrade`, ...) and for each plan to create several `Phases`, `Steps` and `Tasks`.
+
+Also, any Framework will have a `Task` in the `deploy` plan to create similar Kubernetes objects (`StatefulSets`, `Services`, ...).
+
+A Skeleton generator could create `yaml` templates with the common `Plans`, `Phases`, `Steps` and `Tasks`.
```

Comment:
I think the tool should be part of our CLI, rather than a new tool.

Reply:
+1